cloud/services/container/clusters/reconcile.go

```diff
@@ -268,6 +268,9 @@ func (s *Service) createCluster(ctx context.Context, log *logr.Logger) error {
 	if !s.scope.IsAutopilotCluster() {
 		cluster.NodePools = scope.ConvertToSdkNodePools(nodePools, machinePools, isRegional, cluster.Name)
 	}
+	if s.scope.GCPManagedControlPlane.Spec.NetworkPolicy != nil {
+		cluster.NetworkPolicy = infrav1exp.ConvertToSdkNetworkPolicy(s.scope.GCPManagedControlPlane.Spec.NetworkPolicy)
```

Comment:
The "convert" functions are currently located in the same package as this code or in the "scope" package. Could we follow existing conventions and move the convert function into either of those 2 places?

Reply:
Done